scripts/ci/test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e -o pipefail
+set -x
+RETRY=600
+SLEEP=1
+
+export IMAGE=$image
+export VERSION=$tagci
+
+docker pull $IMAGE:$VERSION
+./docker/run devel up -d 
+
+printf "Waiting for stack to come up "
+while [ $RETRY -ge 0 ]; do
+    if nc -z localhost 2308; then 
```

Comment:
The port is wrong. The correct one is **2380**

Reply:
thanks @felipefss, you are right :)